Test/FMI3/fmi3_import_type_definitions_test.cpp

```diff
@@ -0,0 +1,1393 @@
+#include <stdio.h>
+#include <float.h>
+
+#include <fmilib.h>
+#include "config_test.h"
+#include "fmi_testutil.h"
+
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#define UINTXX_MIN (0)
+
+/* delta offsets from extreme values */
+#define D_TYPE  (1)  /* for types when overriding defaults */
+#define D_VAR   (2)  /* for variables when overriding defaults/typedefs */
+#define D_START (3)  /* for variable start values when overriding defaults */
+
+static int G_nErrors; /* global used to count the number of errors reported to the logger during parsing - reset before use! */
+
+static fmi3_import_variable_typedef_t* get_typedef_by_name(fmi3_import_type_definitions_t* tds, const char* name) {
```

Comment:
Not sure how useful this would be in practice, but this almost feels like something that could also be part of the API.

Reply:
I don't know either. But I don't want to push for new API unless requested or if it would be a large performance gain

Reply:
Ok.